projects/objects/factory/conveyors/protos/ConveyorPlatform.proto

```diff
@@ -0,0 +1,650 @@
+#VRML_SIM R2020a utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0.
+# This is a PROTO file for Webots of a conveyor platform for the robot Robotino 3 from Festo.
+# It conveys at a constant speed to fullfil your requirement.
+# Use it with the "extension" field of the Robotino 3 platform.
```

Comment:
```suggestion
# The conveyor contains 3 controllable LEDs.
```
robotino should not be mentioned here are we want this platform to be more generic, but we can mention in the robotino that this platform might be added in the extension slot.